common/app/views/main.scala.html

```diff
@@ -25,6 +25,8 @@
             ("childrens-books-site", metaData.section == "childrens-books-site")))"
         itemscope itemtype="http://schema.org/WebPage">
 
+        <div class="win-chrome-font-fix">@* hack to fix fonts in win/chrome with font-smoothing disabled *@</div>
```

Comment:
As far as I know hinting is not affecting this issue. There is only one instance when hinting fixes horrible rendering - when there is no display driver (graphics acceleration) and Firefox is used (on all Win versions). Apart from that hinting sometimes looks better and sometimes worse but never fixes unreadable font rendering. Screenshots showing different hinting/browser/font smoothing options combinations are on our Win7 test machine.

If there would be similar fix for FF (to force font smoothing when the setting for that is off in Windows) it would be nice to use it (IE completely disregards system font smoothing settings :).